passport-config.js

```diff
@@ -0,0 +1,31 @@
+const GitHubStrategy = require("passport-github").Strategy;
+if (process.env.NODE_ENV !== "production") {
+  require("dotenv").config();
+}
+
+function initialize(passport, users) {
+  passport.use(
+    new GitHubStrategy(
+      {
+        clientID: process.env.GITHUB_CLIENT_ID,
+        clientSecret: process.env.GITHUB_CLIENT_SECRET,
+        callbackURL: "http://mineflayer.cap.lukec.me/auth/github/callback",
```

Comment:
can we store this an environ var? Would be a lot better than manually encoding the URL, in case others want to run it or we want to put it on a different URL

Reply:
that’s actually a really good idea considering how much trouble that var gave me